[0m13:59:31.144966 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f14395340b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1439c26e70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1439535490>]}


============================== 13:59:31.150348 | 6200586c-6edd-417c-904c-d9ba7d251db9 ==============================
[0m13:59:31.150348 [info ] [MainThread]: Running with dbt=1.8.7
[0m13:59:31.151003 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt init dbt_hevo', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m13:59:31.170910 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object, got 'PosixPath'.. Dictionary: {'dir': PosixPath('/home/ubuntu/.dbt')}
[0m13:59:31.171559 [info ] [MainThread]: Creating dbt configuration folder at 
[0m13:59:31.172933 [debug] [MainThread]: Starter project path: /home/ubuntu/dbt-env/lib/python3.12/site-packages/dbt/include/starter_project
[0m13:59:31.175204 [info ] [MainThread]: 
Your new dbt project "dbt_hevo" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m13:59:31.175629 [info ] [MainThread]: Setting up your profile.
[0m14:10:28.038910 [info ] [MainThread]: Profile dbt_hevo written to /home/ubuntu/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m14:10:28.040989 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 656.95465, "process_user_time": 2.035077, "process_kernel_time": 0.186664, "process_mem_max_rss": "108032", "process_in_blocks": "20008", "process_out_blocks": "80"}
[0m14:10:28.041640 [debug] [MainThread]: Command `dbt init` succeeded at 14:10:28.041479 after 656.96 seconds
[0m14:10:28.042020 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1438ece840>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1437797aa0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f143983bef0>]}
[0m14:10:28.042420 [debug] [MainThread]: Flushing usage events
[0m14:11:04.415330 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3e603aee70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3e5f20e8a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3e5f20ec00>]}


============================== 14:11:04.423412 | 9e504032-f7bc-4afb-8103-9797d66985ba ==============================
[0m14:11:04.423412 [info ] [MainThread]: Running with dbt=1.8.7
[0m14:11:04.424337 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'invocation_command': 'dbt debug', 'send_anonymous_usage_stats': 'True'}
[0m14:11:04.442347 [info ] [MainThread]: dbt version: 1.8.7
[0m14:11:04.443333 [info ] [MainThread]: python version: 3.12.3
[0m14:11:04.443790 [info ] [MainThread]: python path: /home/ubuntu/dbt-env/bin/python3
[0m14:11:04.444143 [info ] [MainThread]: os info: Linux-6.8.0-1016-aws-x86_64-with-glibc2.39
[0m14:11:04.750498 [info ] [MainThread]: Using profiles dir at /home/ubuntu/.dbt
[0m14:11:04.751191 [info ] [MainThread]: Using profiles.yml file at /home/ubuntu/.dbt/profiles.yml
[0m14:11:04.751550 [info ] [MainThread]: Using dbt_project.yml file at /home/ubuntu/HEVO_POC_STEPS/dbt_project.yml
[0m14:11:04.753952 [info ] [MainThread]: adapter type: snowflake
[0m14:11:04.754342 [info ] [MainThread]: adapter version: 1.8.4
[0m14:11:04.754684 [info ] [MainThread]: Configuration:
[0m14:11:04.754992 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m14:11:04.755436 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m14:11:04.755799 [info ] [MainThread]: Required dependencies:
[0m14:11:04.756239 [debug] [MainThread]: Executing "git --help"
[0m14:11:04.760085 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--config-env=<name>=<envvar>] <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m14:11:04.760520 [debug] [MainThread]: STDERR: "b''"
[0m14:11:04.760801 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m14:11:04.761153 [info ] [MainThread]: Connection:
[0m14:11:04.761529 [info ] [MainThread]:   account: xnb40345.us-east-1
[0m14:11:04.761916 [info ] [MainThread]:   user: piyushasahni
[0m14:11:04.762368 [info ] [MainThread]:   database: HEVO_DB
[0m14:11:04.762723 [info ] [MainThread]:   warehouse: COMPUTE_WH
[0m14:11:04.763139 [info ] [MainThread]:   role: ACCOUNTADMIN
[0m14:11:04.763498 [info ] [MainThread]:   schema: HEVO_SCHEM
[0m14:11:04.763823 [info ] [MainThread]:   authenticator: None
[0m14:11:04.764210 [info ] [MainThread]:   oauth_client_id: None
[0m14:11:04.764575 [info ] [MainThread]:   query_tag: None
[0m14:11:04.764900 [info ] [MainThread]:   client_session_keep_alive: False
[0m14:11:04.765336 [info ] [MainThread]:   host: None
[0m14:11:04.765790 [info ] [MainThread]:   port: None
[0m14:11:04.766130 [info ] [MainThread]:   proxy_host: None
[0m14:11:04.766476 [info ] [MainThread]:   proxy_port: None
[0m14:11:04.766821 [info ] [MainThread]:   protocol: None
[0m14:11:04.767142 [info ] [MainThread]:   connect_retries: 1
[0m14:11:04.767475 [info ] [MainThread]:   connect_timeout: None
[0m14:11:04.767804 [info ] [MainThread]:   retry_on_database_errors: False
[0m14:11:04.768168 [info ] [MainThread]:   retry_all: False
[0m14:11:04.768521 [info ] [MainThread]:   insecure_mode: False
[0m14:11:04.768845 [info ] [MainThread]:   reuse_connections: None
[0m14:11:04.769343 [info ] [MainThread]: Registered adapter: snowflake=1.8.4
[0m14:11:04.860377 [debug] [MainThread]: Acquiring new snowflake connection 'debug'
[0m14:11:04.861149 [debug] [MainThread]: Using snowflake connection "debug"
[0m14:11:04.861485 [debug] [MainThread]: On debug: select 1 as id
[0m14:11:04.861774 [debug] [MainThread]: Opening a new connection, currently in state init
[0m14:11:06.350238 [debug] [MainThread]: SQL status: SUCCESS 1 in 1.488 seconds
[0m14:11:06.351528 [debug] [MainThread]: On debug: Close
[0m14:11:06.636928 [info ] [MainThread]:   Connection test: [[32mOK connection ok[0m]

[0m14:11:06.637635 [info ] [MainThread]: [31m1 check failed:[0m
[0m14:11:06.638942 [info ] [MainThread]: Project loading failed for the following reason:
 project path </home/ubuntu/HEVO_POC_STEPS/dbt_project.yml> not found

[0m14:11:06.642618 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_wall_clock_time": 2.2805154, "process_user_time": 2.165222, "process_kernel_time": 0.166986, "process_mem_max_rss": "116524", "process_in_blocks": "336", "process_out_blocks": "1216", "command_success": false}
[0m14:11:06.643917 [debug] [MainThread]: Command `dbt debug` failed at 14:11:06.643791 after 2.28 seconds
[0m14:11:06.645862 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m14:11:06.646247 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3e603aee70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3e5d579370>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3e5e258770>]}
[0m14:11:06.647104 [debug] [MainThread]: Flushing usage events
[0m14:13:06.410364 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7b31047cbe30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7b31040d67e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7b31040d7bf0>]}


============================== 14:13:06.414392 | 380c4dc8-f87f-4b86-b5c7-7d8028e52f42 ==============================
[0m14:13:06.414392 [info ] [MainThread]: Running with dbt=1.8.7
[0m14:13:06.415104 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'version_check': 'True', 'debug': 'False', 'log_path': '/home/ubuntu/HEVO_POC_STEPS/logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'invocation_command': 'dbt debug', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m14:13:06.429257 [info ] [MainThread]: dbt version: 1.8.7
[0m14:13:06.429847 [info ] [MainThread]: python version: 3.12.3
[0m14:13:06.430478 [info ] [MainThread]: python path: /home/ubuntu/dbt-env/bin/python3
[0m14:13:06.430848 [info ] [MainThread]: os info: Linux-6.8.0-1016-aws-x86_64-with-glibc2.39
[0m14:13:06.769614 [info ] [MainThread]: Using profiles dir at /home/ubuntu/.dbt
[0m14:13:06.770361 [info ] [MainThread]: Using profiles.yml file at /home/ubuntu/.dbt/profiles.yml
[0m14:13:06.770760 [info ] [MainThread]: Using dbt_project.yml file at /home/ubuntu/HEVO_POC_STEPS/dbt_project.yml
[0m14:13:06.771299 [info ] [MainThread]: adapter type: snowflake
[0m14:13:06.771681 [info ] [MainThread]: adapter version: 1.8.4
[0m14:13:06.871957 [info ] [MainThread]: Configuration:
[0m14:13:06.872538 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m14:13:06.872920 [info ] [MainThread]:   dbt_project.yml file [[32mOK found and valid[0m]
[0m14:13:06.873369 [info ] [MainThread]: Required dependencies:
[0m14:13:06.873833 [debug] [MainThread]: Executing "git --help"
[0m14:13:06.875988 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--config-env=<name>=<envvar>] <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m14:13:06.876412 [debug] [MainThread]: STDERR: "b''"
[0m14:13:06.876699 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m14:13:06.877050 [info ] [MainThread]: Connection:
[0m14:13:06.877495 [info ] [MainThread]:   account: xnb40345.us-east-1
[0m14:13:06.877922 [info ] [MainThread]:   user: piyushasahni
[0m14:13:06.878265 [info ] [MainThread]:   database: HEVO_DB
[0m14:13:06.878632 [info ] [MainThread]:   warehouse: COMPUTE_WH
[0m14:13:06.879064 [info ] [MainThread]:   role: ACCOUNTADMIN
[0m14:13:06.879453 [info ] [MainThread]:   schema: HEVO_SCHEM
[0m14:13:06.879859 [info ] [MainThread]:   authenticator: None
[0m14:13:06.880519 [info ] [MainThread]:   oauth_client_id: None
[0m14:13:06.880917 [info ] [MainThread]:   query_tag: None
[0m14:13:06.881256 [info ] [MainThread]:   client_session_keep_alive: False
[0m14:13:06.881630 [info ] [MainThread]:   host: None
[0m14:13:06.881959 [info ] [MainThread]:   port: None
[0m14:13:06.882278 [info ] [MainThread]:   proxy_host: None
[0m14:13:06.882628 [info ] [MainThread]:   proxy_port: None
[0m14:13:06.882945 [info ] [MainThread]:   protocol: None
[0m14:13:06.883256 [info ] [MainThread]:   connect_retries: 1
[0m14:13:06.883605 [info ] [MainThread]:   connect_timeout: None
[0m14:13:06.883992 [info ] [MainThread]:   retry_on_database_errors: False
[0m14:13:06.884342 [info ] [MainThread]:   retry_all: False
[0m14:13:06.884670 [info ] [MainThread]:   insecure_mode: False
[0m14:13:06.884989 [info ] [MainThread]:   reuse_connections: None
[0m14:13:06.885494 [info ] [MainThread]: Registered adapter: snowflake=1.8.4
[0m14:13:06.977252 [debug] [MainThread]: Acquiring new snowflake connection 'debug'
[0m14:13:06.978189 [debug] [MainThread]: Using snowflake connection "debug"
[0m14:13:06.978549 [debug] [MainThread]: On debug: select 1 as id
[0m14:13:06.978946 [debug] [MainThread]: Opening a new connection, currently in state init
[0m14:13:07.762098 [debug] [MainThread]: SQL status: SUCCESS 1 in 0.783 seconds
[0m14:13:07.763534 [debug] [MainThread]: On debug: Close
[0m14:13:08.221571 [info ] [MainThread]:   Connection test: [[32mOK connection ok[0m]

[0m14:13:08.222290 [info ] [MainThread]: [32mAll checks passed![0m
[0m14:13:08.223522 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": true, "command_wall_clock_time": 1.8881534, "process_user_time": 2.420228, "process_kernel_time": 0.187521, "process_mem_max_rss": "116616", "process_out_blocks": "32", "process_in_blocks": "0"}
[0m14:13:08.224061 [debug] [MainThread]: Command `dbt debug` succeeded at 14:13:08.223950 after 1.89 seconds
[0m14:13:08.224555 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m14:13:08.224893 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7b31047c95b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7b3103ad1340>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7b3103560290>]}
[0m14:13:08.225477 [debug] [MainThread]: Flushing usage events
[0m14:45:19.379314 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76cd949ab8c0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76cd94700ec0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76cd941e77d0>]}


============================== 14:45:19.382774 | af8b2f3b-1ca7-4639-8f06-abd94723ee64 ==============================
[0m14:45:19.382774 [info ] [MainThread]: Running with dbt=1.8.7
[0m14:45:19.383420 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': '/home/ubuntu/HEVO_POC_STEPS/logs', 'version_check': 'True', 'profiles_dir': '/home/ubuntu/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m14:45:19.808601 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'project_id', 'label': 'af8b2f3b-1ca7-4639-8f06-abd94723ee64', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76cd956370e0>]}
[0m14:45:19.868783 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'adapter_info', 'label': 'af8b2f3b-1ca7-4639-8f06-abd94723ee64', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76cd92aec6e0>]}
[0m14:45:19.869673 [info ] [MainThread]: Registered adapter: snowflake=1.8.4
[0m14:45:19.973818 [debug] [MainThread]: checksum: 4af21dafb485259c48497ac86b711ddb1982f3d0f1c0ca4e09356de488b753c0, vars: {}, profile: , target: , version: 1.8.7
[0m14:45:19.974572 [info ] [MainThread]: Unable to do partial parsing because saved manifest not found. Starting full parse.
[0m14:45:19.975011 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'partial_parser', 'label': 'af8b2f3b-1ca7-4639-8f06-abd94723ee64', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76cd929e2e70>]}
[0m14:45:20.799487 [warn ] [MainThread]: [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models.dbt_hevo
[0m14:45:20.804909 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'load_project', 'label': 'af8b2f3b-1ca7-4639-8f06-abd94723ee64', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76cd92763530>]}
[0m14:45:20.865500 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'resource_counts', 'label': 'af8b2f3b-1ca7-4639-8f06-abd94723ee64', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76cd926e9a30>]}
[0m14:45:20.866042 [info ] [MainThread]: Found 459 macros
[0m14:45:20.867812 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'runnable_timing', 'label': 'af8b2f3b-1ca7-4639-8f06-abd94723ee64', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76cd929e0e90>]}
[0m14:45:20.869664 [info ] [MainThread]: 
[0m14:45:20.870238 [warn ] [MainThread]: Nothing to do. Try checking your model configs and model specification args
[0m14:45:20.871000 [debug] [MainThread]: Command end result
[0m14:45:20.963614 [debug] [MainThread]: Resource report: {"command_name": "run", "command_success": true, "command_wall_clock_time": 1.643596, "process_user_time": 3.012657, "process_kernel_time": 0.168274, "process_mem_max_rss": "114660", "process_in_blocks": "96", "process_out_blocks": "2688"}
[0m14:45:20.964228 [debug] [MainThread]: Command `dbt run` succeeded at 14:45:20.964106 after 1.64 seconds
[0m14:45:20.964623 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76cd949ab8c0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76cd979b31d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76cd9280c260>]}
[0m14:45:20.964974 [debug] [MainThread]: Flushing usage events
[0m14:47:44.049247 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76aa624b3950>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76aa62751910>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76aa62737530>]}


============================== 14:47:44.052520 | 2438265e-7047-41c0-aca6-758622e67dbc ==============================
[0m14:47:44.052520 [info ] [MainThread]: Running with dbt=1.8.7
[0m14:47:44.053330 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': '/home/ubuntu/HEVO_POC_STEPS/logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m14:47:44.481546 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'project_id', 'label': '2438265e-7047-41c0-aca6-758622e67dbc', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76aa6044f440>]}
[0m14:47:44.545056 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'adapter_info', 'label': '2438265e-7047-41c0-aca6-758622e67dbc', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76aa608c9fd0>]}
[0m14:47:44.546449 [info ] [MainThread]: Registered adapter: snowflake=1.8.4
[0m14:47:44.645442 [debug] [MainThread]: checksum: 4af21dafb485259c48497ac86b711ddb1982f3d0f1c0ca4e09356de488b753c0, vars: {}, profile: , target: , version: 1.8.7
[0m14:47:44.717170 [debug] [MainThread]: Partial parsing enabled: 0 files deleted, 1 files added, 0 files changed.
[0m14:47:44.717771 [debug] [MainThread]: Partial parsing: added file: dbt_hevo://models/customer_aggregation.sql
[0m14:47:44.959591 [error] [MainThread]: Encountered an error:
Compilation Error
  Model 'model.dbt_hevo.customer_aggregation' (models/customer_aggregation.sql) depends on a node named 'raw_customers' which was not found
[0m14:47:44.960673 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.9709887, "process_user_time": 2.296958, "process_kernel_time": 0.145283, "process_mem_max_rss": "116868", "process_out_blocks": "16", "command_success": false, "process_in_blocks": "0"}
[0m14:47:44.961384 [debug] [MainThread]: Command `dbt run` failed at 14:47:44.961225 after 0.97 seconds
[0m14:47:44.961807 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76aa61ceb980>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76aa63036390>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x76aa604b2b10>]}
[0m14:47:44.962180 [debug] [MainThread]: Flushing usage events
[0m15:00:11.492853 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x700f4d1b9e20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x700f4cc5bdd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x700f4e608c50>]}


============================== 15:00:11.497872 | 4b47ca6d-db06-4d6d-983c-32266db79da1 ==============================
[0m15:00:11.497872 [info ] [MainThread]: Running with dbt=1.8.7
[0m15:00:11.498798 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'version_check': 'True', 'fail_fast': 'False', 'log_path': '/home/ubuntu/HEVO_POC_STEPS/logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt run', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m15:00:11.994773 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'project_id', 'label': '4b47ca6d-db06-4d6d-983c-32266db79da1', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x700f4e242ff0>]}
[0m15:00:12.064603 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'adapter_info', 'label': '4b47ca6d-db06-4d6d-983c-32266db79da1', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x700f4d0caae0>]}
[0m15:00:12.065727 [info ] [MainThread]: Registered adapter: snowflake=1.8.4
[0m15:00:12.182102 [debug] [MainThread]: checksum: 4af21dafb485259c48497ac86b711ddb1982f3d0f1c0ca4e09356de488b753c0, vars: {}, profile: , target: , version: 1.8.7
[0m15:00:12.251612 [debug] [MainThread]: Partial parsing enabled: 0 files deleted, 2 files added, 0 files changed.
[0m15:00:12.252174 [debug] [MainThread]: Partial parsing: added file: dbt_hevo://models/customer_aggregation.sql
[0m15:00:12.252594 [debug] [MainThread]: Partial parsing: added file: dbt_hevo://models/schema.yml
[0m15:00:12.600050 [error] [MainThread]: Encountered an error:
Compilation Error
  Model 'model.dbt_hevo.customer_aggregation' (models/customer_aggregation.sql) depends on a node named 'raw_customers' which was not found
[0m15:00:12.601381 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 1.1820012, "process_user_time": 2.888622, "process_kernel_time": 0.166466, "process_mem_max_rss": "117924", "process_out_blocks": "16", "command_success": false, "process_in_blocks": "0"}
[0m15:00:12.602023 [debug] [MainThread]: Command `dbt run` failed at 15:00:12.601904 after 1.18 seconds
[0m15:00:12.602426 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x700f4cda7bc0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x700f4b21e420>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x700f4b1ea330>]}
[0m15:00:12.602797 [debug] [MainThread]: Flushing usage events
[0m15:01:18.153840 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x755bc1919850>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x755bc3a948f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x755bc2207fe0>]}


============================== 15:01:18.157080 | 16a8c4c3-0e01-41ca-9c44-c9a19be40ea9 ==============================
[0m15:01:18.157080 [info ] [MainThread]: Running with dbt=1.8.7
[0m15:01:18.157848 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'version_check': 'True', 'fail_fast': 'False', 'log_path': '/home/ubuntu/HEVO_POC_STEPS/logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'invocation_command': 'dbt source freshness', 'log_format': 'default', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m15:01:18.630225 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'project_id', 'label': '16a8c4c3-0e01-41ca-9c44-c9a19be40ea9', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x755bc01fa870>]}
[0m15:01:18.707067 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'adapter_info', 'label': '16a8c4c3-0e01-41ca-9c44-c9a19be40ea9', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x755bc018c1d0>]}
[0m15:01:18.708000 [info ] [MainThread]: Registered adapter: snowflake=1.8.4
[0m15:01:18.818773 [debug] [MainThread]: checksum: 4af21dafb485259c48497ac86b711ddb1982f3d0f1c0ca4e09356de488b753c0, vars: {}, profile: , target: , version: 1.8.7
[0m15:01:18.892173 [debug] [MainThread]: Partial parsing enabled: 0 files deleted, 2 files added, 0 files changed.
[0m15:01:18.892835 [debug] [MainThread]: Partial parsing: added file: dbt_hevo://models/customer_aggregation.sql
[0m15:01:18.893331 [debug] [MainThread]: Partial parsing: added file: dbt_hevo://models/schema.yml
[0m15:01:19.297789 [error] [MainThread]: Encountered an error:
Compilation Error
  Model 'model.dbt_hevo.customer_aggregation' (models/customer_aggregation.sql) depends on a node named 'raw_customers' which was not found
[0m15:01:19.298957 [debug] [MainThread]: Resource report: {"command_name": "freshness", "command_wall_clock_time": 1.2106323, "process_user_time": 2.733205, "process_kernel_time": 0.17869, "process_mem_max_rss": "117880", "process_out_blocks": "16", "command_success": false, "process_in_blocks": "0"}
[0m15:01:19.299693 [debug] [MainThread]: Command `dbt source freshness` failed at 15:01:19.299562 after 1.21 seconds
[0m15:01:19.300087 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x755bc1993e60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x755bbfdc9160>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x755bc00b1700>]}
[0m15:01:19.300469 [debug] [MainThread]: Flushing usage events
[0m15:17:59.678296 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7067e1ad5c70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7067e11af710>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7067e11cbc50>]}


============================== 15:17:59.682090 | ba4f1a90-0e58-4365-b930-e528bdb5eca6 ==============================
[0m15:17:59.682090 [info ] [MainThread]: Running with dbt=1.8.7
[0m15:17:59.682689 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'log_path': '/home/ubuntu/HEVO_POC_STEPS/logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m15:18:00.167213 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'project_id', 'label': 'ba4f1a90-0e58-4365-b930-e528bdb5eca6', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7067dfb26e70>]}
[0m15:18:00.231453 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'adapter_info', 'label': 'ba4f1a90-0e58-4365-b930-e528bdb5eca6', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7067dfdcf620>]}
[0m15:18:00.232327 [info ] [MainThread]: Registered adapter: snowflake=1.8.4
[0m15:18:00.334729 [debug] [MainThread]: checksum: 4af21dafb485259c48497ac86b711ddb1982f3d0f1c0ca4e09356de488b753c0, vars: {}, profile: , target: , version: 1.8.7
[0m15:18:00.395617 [debug] [MainThread]: Partial parsing enabled: 0 files deleted, 2 files added, 0 files changed.
[0m15:18:00.396214 [debug] [MainThread]: Partial parsing: added file: dbt_hevo://models/schema.yml
[0m15:18:00.396599 [debug] [MainThread]: Partial parsing: added file: dbt_hevo://models/customer_aggregation.sql
[0m15:18:00.815048 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'load_project', 'label': 'ba4f1a90-0e58-4365-b930-e528bdb5eca6', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7067df5f0bc0>]}
[0m15:18:00.932996 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'resource_counts', 'label': 'ba4f1a90-0e58-4365-b930-e528bdb5eca6', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7067df78b7a0>]}
[0m15:18:00.933824 [info ] [MainThread]: Found 1 model, 3 sources, 459 macros
[0m15:18:00.934250 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'runnable_timing', 'label': 'ba4f1a90-0e58-4365-b930-e528bdb5eca6', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7067df9f2090>]}
[0m15:18:00.937095 [info ] [MainThread]: 
[0m15:18:00.937816 [debug] [MainThread]: Acquiring new snowflake connection 'master'
[0m15:18:00.938912 [debug] [ThreadPool]: Acquiring new snowflake connection 'list_HEVO_DB'
[0m15:18:00.959031 [debug] [ThreadPool]: Using snowflake connection "list_HEVO_DB"
[0m15:18:00.959554 [debug] [ThreadPool]: On list_HEVO_DB: /* {"app": "dbt", "dbt_version": "1.8.7", "profile_name": "dbt_hevo", "target_name": "dev", "connection_name": "list_HEVO_DB"} */
show terse schemas in database HEVO_DB
    limit 10000
[0m15:18:00.959882 [debug] [ThreadPool]: Opening a new connection, currently in state init
[0m15:18:01.826166 [debug] [ThreadPool]: SQL status: SUCCESS 3 in 0.866 seconds
[0m15:18:01.829979 [debug] [ThreadPool]: On list_HEVO_DB: Close
[0m15:18:02.103296 [debug] [ThreadPool]: Re-using an available connection from the pool (formerly list_HEVO_DB, now create_HEVO_DB_HEVO_SCHEM)
[0m15:18:02.104282 [debug] [ThreadPool]: Creating schema "database: "HEVO_DB"
schema: "HEVO_SCHEM"
"
[0m15:18:02.111987 [debug] [ThreadPool]: Using snowflake connection "create_HEVO_DB_HEVO_SCHEM"
[0m15:18:02.112554 [debug] [ThreadPool]: On create_HEVO_DB_HEVO_SCHEM: /* {"app": "dbt", "dbt_version": "1.8.7", "profile_name": "dbt_hevo", "target_name": "dev", "connection_name": "create_HEVO_DB_HEVO_SCHEM"} */
create schema if not exists HEVO_DB.HEVO_SCHEM
[0m15:18:02.112982 [debug] [ThreadPool]: Opening a new connection, currently in state closed
[0m15:18:03.075654 [debug] [ThreadPool]: SQL status: SUCCESS 1 in 0.963 seconds
[0m15:18:03.076920 [debug] [ThreadPool]: On create_HEVO_DB_HEVO_SCHEM: Close
[0m15:18:03.389984 [debug] [ThreadPool]: Acquiring new snowflake connection 'list_HEVO_DB_HEVO_SCHEM'
[0m15:18:03.412628 [debug] [ThreadPool]: Using snowflake connection "list_HEVO_DB_HEVO_SCHEM"
[0m15:18:03.413965 [debug] [ThreadPool]: On list_HEVO_DB_HEVO_SCHEM: /* {"app": "dbt", "dbt_version": "1.8.7", "profile_name": "dbt_hevo", "target_name": "dev", "connection_name": "list_HEVO_DB_HEVO_SCHEM"} */
show objects in HEVO_DB.HEVO_SCHEM limit 10000
[0m15:18:03.414328 [debug] [ThreadPool]: Opening a new connection, currently in state init
[0m15:18:04.069208 [debug] [ThreadPool]: SQL status: SUCCESS 0 in 0.655 seconds
[0m15:18:04.071040 [debug] [ThreadPool]: On list_HEVO_DB_HEVO_SCHEM: Close
[0m15:18:04.339806 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'runnable_timing', 'label': 'ba4f1a90-0e58-4365-b930-e528bdb5eca6', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7067e0d05ca0>]}
[0m15:18:04.340661 [info ] [MainThread]: Concurrency: 1 threads (target='dev')
[0m15:18:04.341095 [info ] [MainThread]: 
[0m15:18:04.345518 [debug] [Thread-1 (]: Began running node model.dbt_hevo.customer_aggregation
[0m15:18:04.346490 [info ] [Thread-1 (]: 1 of 1 START sql view model HEVO_SCHEM.customer_aggregation .................... [RUN]
[0m15:18:04.347042 [debug] [Thread-1 (]: Re-using an available connection from the pool (formerly list_HEVO_DB_HEVO_SCHEM, now model.dbt_hevo.customer_aggregation)
[0m15:18:04.347565 [debug] [Thread-1 (]: Began compiling node model.dbt_hevo.customer_aggregation
[0m15:18:04.360619 [debug] [Thread-1 (]: Writing injected SQL for node "model.dbt_hevo.customer_aggregation"
[0m15:18:04.361497 [debug] [Thread-1 (]: Began executing node model.dbt_hevo.customer_aggregation
[0m15:18:04.404439 [debug] [Thread-1 (]: Writing runtime sql for node "model.dbt_hevo.customer_aggregation"
[0m15:18:04.406937 [debug] [Thread-1 (]: Using snowflake connection "model.dbt_hevo.customer_aggregation"
[0m15:18:04.407555 [debug] [Thread-1 (]: On model.dbt_hevo.customer_aggregation: /* {"app": "dbt", "dbt_version": "1.8.7", "profile_name": "dbt_hevo", "target_name": "dev", "node_id": "model.dbt_hevo.customer_aggregation"} */
create or replace   view HEVO_DB.HEVO_SCHEM.customer_aggregation
  
   as (
    WITH customer_orders AS (
    SELECT
        c.id AS customer_id,
        c.first_name,
        c.last_name,
        MIN(o.order_date) AS first_order,
        MAX(o.order_date) AS most_recent_order,
        COUNT(o.id) AS number_of_orders
    FROM HEVO_DB.HEVO_SCHEMA.raw_customers c
    LEFT JOIN HEVO_DB.HEVO_SCHEMA.raw_orders o
    ON c.id = o.user_id
    GROUP BY c.id, c.first_name, c.last_name
),
customer_lifetime_value AS (
    SELECT
        o.user_id AS customer_id,
        SUM(p.amount) AS customer_lifetime_value
    FROM HEVO_DB.HEVO_SCHEMA.raw_orders o
    LEFT JOIN HEVO_DB.HEVO_SCHEMA.raw_payments p
    ON o.id = p.order_id
    GROUP BY o.user_id
)
SELECT
    co.customer_id,
    co.first_name,
    co.last_name,
    co.first_order,
    co.most_recent_order,
    co.number_of_orders,
    clv.customer_lifetime_value
FROM customer_orders co
LEFT JOIN customer_lifetime_value clv
ON co.customer_id = clv.customer_id;
[0m15:18:04.408223 [debug] [Thread-1 (]: Opening a new connection, currently in state closed
[0m15:18:05.192148 [debug] [Thread-1 (]: Snowflake adapter: Snowflake query id: 01b874b6-010b-7533-0009-daff00017056
[0m15:18:05.192926 [debug] [Thread-1 (]: Snowflake adapter: Snowflake error: 001003 (42000): SQL compilation error:
syntax error line 36 at position 35 unexpected ';'.
[0m15:18:05.193460 [debug] [Thread-1 (]: On model.dbt_hevo.customer_aggregation: Close
[0m15:18:05.464149 [debug] [Thread-1 (]: Database Error in model customer_aggregation (models/customer_aggregation.sql)
  001003 (42000): SQL compilation error:
  syntax error line 36 at position 35 unexpected ';'.
  compiled code at target/run/dbt_hevo/models/customer_aggregation.sql
[0m15:18:05.466009 [debug] [Thread-1 (]: Sending event: {'category': 'dbt', 'action': 'run_model', 'label': 'ba4f1a90-0e58-4365-b930-e528bdb5eca6', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7067e1938ad0>]}
[0m15:18:05.466858 [error] [Thread-1 (]: 1 of 1 ERROR creating sql view model HEVO_SCHEM.customer_aggregation ........... [[31mERROR[0m in 1.12s]
[0m15:18:05.467587 [debug] [Thread-1 (]: Finished running node model.dbt_hevo.customer_aggregation
[0m15:18:05.469843 [debug] [MainThread]: Connection 'master' was properly closed.
[0m15:18:05.470156 [debug] [MainThread]: Connection 'create_HEVO_DB_HEVO_SCHEM' was properly closed.
[0m15:18:05.470457 [debug] [MainThread]: Connection 'model.dbt_hevo.customer_aggregation' was properly closed.
[0m15:18:05.470856 [info ] [MainThread]: 
[0m15:18:05.471252 [info ] [MainThread]: Finished running 1 view model in 0 hours 0 minutes and 4.53 seconds (4.53s).
[0m15:18:05.472020 [debug] [MainThread]: Command end result
[0m15:18:05.606505 [info ] [MainThread]: 
[0m15:18:05.607223 [info ] [MainThread]: [31mCompleted with 1 error and 0 warnings:[0m
[0m15:18:05.607789 [info ] [MainThread]: 
[0m15:18:05.608276 [error] [MainThread]:   Database Error in model customer_aggregation (models/customer_aggregation.sql)
  001003 (42000): SQL compilation error:
  syntax error line 36 at position 35 unexpected ';'.
  compiled code at target/run/dbt_hevo/models/customer_aggregation.sql
[0m15:18:05.608839 [info ] [MainThread]: 
[0m15:18:05.609242 [info ] [MainThread]: Done. PASS=0 WARN=0 ERROR=1 SKIP=0 TOTAL=1
[0m15:18:05.610253 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 5.9900985, "process_user_time": 3.217906, "process_kernel_time": 0.203219, "process_mem_max_rss": "127844", "process_in_blocks": "336", "process_out_blocks": "2768", "command_success": false}
[0m15:18:05.610850 [debug] [MainThread]: Command `dbt run` failed at 15:18:05.610724 after 5.99 seconds
[0m15:18:05.611224 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7067dfa57f80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7067dfc24fb0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7067e014bf20>]}
[0m15:18:05.611621 [debug] [MainThread]: Flushing usage events
[0m15:20:55.355021 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d5fdfb0adb0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d5fdee16c00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d5fdfb0ac90>]}


============================== 15:20:55.369038 | f8329010-6cb0-420e-b241-a6edc24e24f0 ==============================
[0m15:20:55.369038 [info ] [MainThread]: Running with dbt=1.8.7
[0m15:20:55.370120 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': '/home/ubuntu/HEVO_POC_STEPS/logs', 'version_check': 'True', 'profiles_dir': '/home/ubuntu/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m15:20:55.916209 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'project_id', 'label': 'f8329010-6cb0-420e-b241-a6edc24e24f0', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d5fdfc7af90>]}
[0m15:20:56.017569 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'adapter_info', 'label': 'f8329010-6cb0-420e-b241-a6edc24e24f0', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d5fdfb84380>]}
[0m15:20:56.019902 [info ] [MainThread]: Registered adapter: snowflake=1.8.4
[0m15:20:56.183284 [debug] [MainThread]: checksum: 4af21dafb485259c48497ac86b711ddb1982f3d0f1c0ca4e09356de488b753c0, vars: {}, profile: , target: , version: 1.8.7
[0m15:20:56.350810 [debug] [MainThread]: Partial parsing enabled: 0 files deleted, 0 files added, 1 files changed.
[0m15:20:56.351669 [debug] [MainThread]: Partial parsing: updated file: dbt_hevo://models/customer_aggregation.sql
[0m15:20:56.721802 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'load_project', 'label': 'f8329010-6cb0-420e-b241-a6edc24e24f0', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d5fdd11aed0>]}
[0m15:20:56.856967 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'resource_counts', 'label': 'f8329010-6cb0-420e-b241-a6edc24e24f0', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d5fdea57230>]}
[0m15:20:56.858978 [info ] [MainThread]: Found 1 model, 3 sources, 459 macros
[0m15:20:56.861610 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'runnable_timing', 'label': 'f8329010-6cb0-420e-b241-a6edc24e24f0', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d5fdd209a90>]}
[0m15:20:56.866113 [info ] [MainThread]: 
[0m15:20:56.866938 [debug] [MainThread]: Acquiring new snowflake connection 'master'
[0m15:20:56.868386 [debug] [ThreadPool]: Acquiring new snowflake connection 'list_HEVO_DB'
[0m15:20:56.885996 [debug] [ThreadPool]: Using snowflake connection "list_HEVO_DB"
[0m15:20:56.886481 [debug] [ThreadPool]: On list_HEVO_DB: /* {"app": "dbt", "dbt_version": "1.8.7", "profile_name": "dbt_hevo", "target_name": "dev", "connection_name": "list_HEVO_DB"} */
show terse schemas in database HEVO_DB
    limit 10000
[0m15:20:56.886794 [debug] [ThreadPool]: Opening a new connection, currently in state init
[0m15:20:57.847939 [debug] [ThreadPool]: SQL status: SUCCESS 4 in 0.961 seconds
[0m15:20:57.849951 [debug] [ThreadPool]: On list_HEVO_DB: Close
[0m15:20:58.137860 [debug] [ThreadPool]: Acquiring new snowflake connection 'list_HEVO_DB_HEVO_SCHEM'
[0m15:20:58.166064 [debug] [ThreadPool]: Using snowflake connection "list_HEVO_DB_HEVO_SCHEM"
[0m15:20:58.166617 [debug] [ThreadPool]: On list_HEVO_DB_HEVO_SCHEM: /* {"app": "dbt", "dbt_version": "1.8.7", "profile_name": "dbt_hevo", "target_name": "dev", "connection_name": "list_HEVO_DB_HEVO_SCHEM"} */
show objects in HEVO_DB.HEVO_SCHEM limit 10000
[0m15:20:58.166955 [debug] [ThreadPool]: Opening a new connection, currently in state init
[0m15:20:58.942024 [debug] [ThreadPool]: SQL status: SUCCESS 0 in 0.775 seconds
[0m15:20:58.946780 [debug] [ThreadPool]: On list_HEVO_DB_HEVO_SCHEM: Close
[0m15:20:59.225587 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'runnable_timing', 'label': 'f8329010-6cb0-420e-b241-a6edc24e24f0', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d5fdd24d970>]}
[0m15:20:59.226566 [info ] [MainThread]: Concurrency: 1 threads (target='dev')
[0m15:20:59.227462 [info ] [MainThread]: 
[0m15:20:59.231044 [debug] [Thread-1 (]: Began running node model.dbt_hevo.customer_aggregation
[0m15:20:59.232240 [info ] [Thread-1 (]: 1 of 1 START sql view model HEVO_SCHEM.customer_aggregation .................... [RUN]
[0m15:20:59.232893 [debug] [Thread-1 (]: Re-using an available connection from the pool (formerly list_HEVO_DB_HEVO_SCHEM, now model.dbt_hevo.customer_aggregation)
[0m15:20:59.233451 [debug] [Thread-1 (]: Began compiling node model.dbt_hevo.customer_aggregation
[0m15:20:59.249057 [debug] [Thread-1 (]: Writing injected SQL for node "model.dbt_hevo.customer_aggregation"
[0m15:20:59.250617 [debug] [Thread-1 (]: Began executing node model.dbt_hevo.customer_aggregation
[0m15:20:59.296488 [debug] [Thread-1 (]: Writing runtime sql for node "model.dbt_hevo.customer_aggregation"
[0m15:20:59.298666 [debug] [Thread-1 (]: Using snowflake connection "model.dbt_hevo.customer_aggregation"
[0m15:20:59.299232 [debug] [Thread-1 (]: On model.dbt_hevo.customer_aggregation: /* {"app": "dbt", "dbt_version": "1.8.7", "profile_name": "dbt_hevo", "target_name": "dev", "node_id": "model.dbt_hevo.customer_aggregation"} */
create or replace   view HEVO_DB.HEVO_SCHEM.customer_aggregation
  
   as (
    WITH customer_orders AS (
    SELECT
        c.id AS customer_id,
        c.first_name,
        c.last_name,
        MIN(o.order_date) AS first_order,
        MAX(o.order_date) AS most_recent_order,
        COUNT(o.id) AS number_of_orders
    FROM HEVO_DB.HEVO_SCHEMA.raw_customers c
    LEFT JOIN HEVO_DB.HEVO_SCHEMA.raw_orders o
    ON c.id = o.user_id
    GROUP BY c.id, c.first_name, c.last_name
),
customer_lifetime_value AS (
    SELECT
        o.user_id AS customer_id,
        SUM(p.amount) AS customer_lifetime_value
    FROM HEVO_DB.HEVO_SCHEMA.raw_orders o
    LEFT JOIN HEVO_DB.HEVO_SCHEMA.raw_payments p
    ON o.id = p.order_id
    GROUP BY o.user_id
)
SELECT
    co.customer_id,
    co.first_name,
    co.last_name,
    co.first_order,
    co.most_recent_order,
    co.number_of_orders,
    clv.customer_lifetime_value
FROM customer_orders co
LEFT JOIN customer_lifetime_value clv
ON co.customer_id = clv.customer_id
  );
[0m15:20:59.299692 [debug] [Thread-1 (]: Opening a new connection, currently in state closed
[0m15:21:00.470842 [debug] [Thread-1 (]: SQL status: SUCCESS 1 in 1.171 seconds
[0m15:21:00.505608 [debug] [Thread-1 (]: On model.dbt_hevo.customer_aggregation: Close
[0m15:21:00.770761 [debug] [Thread-1 (]: Sending event: {'category': 'dbt', 'action': 'run_model', 'label': 'f8329010-6cb0-420e-b241-a6edc24e24f0', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d5fdf418b00>]}
[0m15:21:00.772178 [info ] [Thread-1 (]: 1 of 1 OK created sql view model HEVO_SCHEM.customer_aggregation ............... [[32mSUCCESS 1[0m in 1.54s]
[0m15:21:00.776443 [debug] [Thread-1 (]: Finished running node model.dbt_hevo.customer_aggregation
[0m15:21:00.779799 [debug] [MainThread]: Connection 'master' was properly closed.
[0m15:21:00.780173 [debug] [MainThread]: Connection 'list_HEVO_DB' was properly closed.
[0m15:21:00.780506 [debug] [MainThread]: Connection 'model.dbt_hevo.customer_aggregation' was properly closed.
[0m15:21:00.780824 [info ] [MainThread]: 
[0m15:21:00.781389 [info ] [MainThread]: Finished running 1 view model in 0 hours 0 minutes and 3.91 seconds (3.91s).
[0m15:21:00.786686 [debug] [MainThread]: Command end result
[0m15:21:00.829270 [info ] [MainThread]: 
[0m15:21:00.830006 [info ] [MainThread]: [32mCompleted successfully[0m
[0m15:21:00.830461 [info ] [MainThread]: 
[0m15:21:00.830897 [info ] [MainThread]: Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[0m15:21:00.831901 [debug] [MainThread]: Resource report: {"command_name": "run", "command_success": true, "command_wall_clock_time": 5.5705533, "process_user_time": 3.480709, "process_kernel_time": 0.220195, "process_mem_max_rss": "126440", "process_out_blocks": "2768", "process_in_blocks": "0"}
[0m15:21:00.832487 [debug] [MainThread]: Command `dbt run` succeeded at 15:21:00.832360 after 5.57 seconds
[0m15:21:00.832870 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d5fdecf8ec0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d5fdecf8ce0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d5fdc835520>]}
[0m15:21:00.833515 [debug] [MainThread]: Flushing usage events
[0m15:25:29.199571 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7058e788d430>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7058e76f8500>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7058e7ac7d40>]}


============================== 15:25:29.202848 | 8b5524f9-55cf-4d04-8429-5ed46f1f905e ==============================
[0m15:25:29.202848 [info ] [MainThread]: Running with dbt=1.8.7
[0m15:25:29.203604 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'version_check': 'True', 'debug': 'False', 'log_path': '/home/ubuntu/HEVO_POC_STEPS/logs', 'warn_error': 'None', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt test', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m15:25:29.609291 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'project_id', 'label': '8b5524f9-55cf-4d04-8429-5ed46f1f905e', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7058e831ef90>]}
[0m15:25:29.669944 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'adapter_info', 'label': '8b5524f9-55cf-4d04-8429-5ed46f1f905e', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7058e6705a00>]}
[0m15:25:29.670803 [info ] [MainThread]: Registered adapter: snowflake=1.8.4
[0m15:25:29.763926 [debug] [MainThread]: checksum: 4af21dafb485259c48497ac86b711ddb1982f3d0f1c0ca4e09356de488b753c0, vars: {}, profile: , target: , version: 1.8.7
[0m15:25:29.877007 [debug] [MainThread]: Partial parsing enabled: 0 files deleted, 0 files added, 0 files changed.
[0m15:25:29.877562 [debug] [MainThread]: Partial parsing enabled, no changes found, skipping parsing
[0m15:25:29.913957 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'load_project', 'label': '8b5524f9-55cf-4d04-8429-5ed46f1f905e', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7058e56c8a10>]}
[0m15:25:30.028526 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'resource_counts', 'label': '8b5524f9-55cf-4d04-8429-5ed46f1f905e', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7058e519e630>]}
[0m15:25:30.029249 [info ] [MainThread]: Found 1 model, 3 sources, 459 macros
[0m15:25:30.029749 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'runnable_timing', 'label': '8b5524f9-55cf-4d04-8429-5ed46f1f905e', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7058e516a630>]}
[0m15:25:30.031444 [info ] [MainThread]: 
[0m15:25:30.031879 [warn ] [MainThread]: Nothing to do. Try checking your model configs and model specification args
[0m15:25:30.034043 [debug] [MainThread]: Command end result
[0m15:25:30.058221 [debug] [MainThread]: Resource report: {"command_name": "test", "command_success": true, "command_wall_clock_time": 0.9163037, "process_user_time": 2.376541, "process_kernel_time": 0.179934, "process_mem_max_rss": "117788", "process_in_blocks": "1080", "process_out_blocks": "1832"}
[0m15:25:30.058935 [debug] [MainThread]: Command `dbt test` succeeded at 15:25:30.058808 after 0.92 seconds
[0m15:25:30.059448 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7058e788d430>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7058e72c22a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7058e700fb60>]}
[0m15:25:30.059834 [debug] [MainThread]: Flushing usage events
[0m15:25:39.380276 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x714cce3c2750>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x714ccead6ba0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x714ccdd8fa40>]}


============================== 15:25:39.384684 | acb652ff-d5c9-4141-b813-7e7c24a745ac ==============================
[0m15:25:39.384684 [info ] [MainThread]: Running with dbt=1.8.7
[0m15:25:39.385369 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': '/home/ubuntu/HEVO_POC_STEPS/logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt docs generate', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m15:25:39.874725 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'project_id', 'label': 'acb652ff-d5c9-4141-b813-7e7c24a745ac', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x714ccd4c0530>]}
[0m15:25:39.958374 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'adapter_info', 'label': 'acb652ff-d5c9-4141-b813-7e7c24a745ac', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x714ccc69f6e0>]}
[0m15:25:39.959384 [info ] [MainThread]: Registered adapter: snowflake=1.8.4
[0m15:25:40.060446 [debug] [MainThread]: checksum: 4af21dafb485259c48497ac86b711ddb1982f3d0f1c0ca4e09356de488b753c0, vars: {}, profile: , target: , version: 1.8.7
[0m15:25:40.158407 [debug] [MainThread]: Partial parsing enabled: 0 files deleted, 0 files added, 0 files changed.
[0m15:25:40.158824 [debug] [MainThread]: Partial parsing enabled, no changes found, skipping parsing
[0m15:25:40.199165 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'load_project', 'label': 'acb652ff-d5c9-4141-b813-7e7c24a745ac', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x714ccd9686e0>]}
[0m15:25:40.211945 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'resource_counts', 'label': 'acb652ff-d5c9-4141-b813-7e7c24a745ac', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x714ccc0106e0>]}
[0m15:25:40.212457 [info ] [MainThread]: Found 1 model, 3 sources, 459 macros
[0m15:25:40.212945 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'runnable_timing', 'label': 'acb652ff-d5c9-4141-b813-7e7c24a745ac', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x714ccc077830>]}
[0m15:25:40.214699 [info ] [MainThread]: 
[0m15:25:40.215330 [debug] [MainThread]: Acquiring new snowflake connection 'master'
[0m15:25:40.216240 [debug] [ThreadPool]: Acquiring new snowflake connection 'list_HEVO_DB_HEVO_SCHEM'
[0m15:25:40.237629 [debug] [ThreadPool]: Using snowflake connection "list_HEVO_DB_HEVO_SCHEM"
[0m15:25:40.238058 [debug] [ThreadPool]: On list_HEVO_DB_HEVO_SCHEM: /* {"app": "dbt", "dbt_version": "1.8.7", "profile_name": "dbt_hevo", "target_name": "dev", "connection_name": "list_HEVO_DB_HEVO_SCHEM"} */
show objects in HEVO_DB.HEVO_SCHEM limit 10000
[0m15:25:40.238368 [debug] [ThreadPool]: Opening a new connection, currently in state init
[0m15:25:41.036882 [debug] [ThreadPool]: SQL status: SUCCESS 1 in 0.798 seconds
[0m15:25:41.039930 [debug] [ThreadPool]: On list_HEVO_DB_HEVO_SCHEM: Close
[0m15:25:41.311247 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'runnable_timing', 'label': 'acb652ff-d5c9-4141-b813-7e7c24a745ac', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x714ccc0f4620>]}
[0m15:25:41.311868 [info ] [MainThread]: Concurrency: 1 threads (target='dev')
[0m15:25:41.312428 [info ] [MainThread]: 
[0m15:25:41.314779 [debug] [Thread-1 (]: Began running node model.dbt_hevo.customer_aggregation
[0m15:25:41.315236 [debug] [Thread-1 (]: Re-using an available connection from the pool (formerly list_HEVO_DB_HEVO_SCHEM, now model.dbt_hevo.customer_aggregation)
[0m15:25:41.315626 [debug] [Thread-1 (]: Began compiling node model.dbt_hevo.customer_aggregation
[0m15:25:41.325025 [debug] [Thread-1 (]: Writing injected SQL for node "model.dbt_hevo.customer_aggregation"
[0m15:25:41.325721 [debug] [Thread-1 (]: Began executing node model.dbt_hevo.customer_aggregation
[0m15:25:41.326505 [debug] [Thread-1 (]: Finished running node model.dbt_hevo.customer_aggregation
[0m15:25:41.327731 [debug] [MainThread]: Connection 'master' was properly closed.
[0m15:25:41.328089 [debug] [MainThread]: Connection 'model.dbt_hevo.customer_aggregation' was properly closed.
[0m15:25:41.329870 [debug] [MainThread]: Command end result
[0m15:25:41.416064 [debug] [MainThread]: Acquiring new snowflake connection 'generate_catalog'
[0m15:25:41.416484 [info ] [MainThread]: Building catalog
[0m15:25:41.427736 [debug] [ThreadPool]: Acquiring new snowflake connection 'HEVO_DB.information_schema'
[0m15:25:41.450804 [debug] [ThreadPool]: Using snowflake connection "HEVO_DB.information_schema"
[0m15:25:41.451432 [debug] [ThreadPool]: On HEVO_DB.information_schema: /* {"app": "dbt", "dbt_version": "1.8.7", "profile_name": "dbt_hevo", "target_name": "dev", "connection_name": "HEVO_DB.information_schema"} */
with tables as (
            select
        table_catalog as "table_database",
        table_schema as "table_schema",
        table_name as "table_name",
        case
            when is_dynamic = 'YES' and table_type = 'BASE TABLE' THEN 'DYNAMIC TABLE'
            else table_type
        end as "table_type",
        comment as "table_comment",

        -- note: this is the _role_ that owns the table
        table_owner as "table_owner",

        'Clustering Key' as "stats:clustering_key:label",
        clustering_key as "stats:clustering_key:value",
        'The key used to cluster this table' as "stats:clustering_key:description",
        (clustering_key is not null) as "stats:clustering_key:include",

        'Row Count' as "stats:row_count:label",
        row_count as "stats:row_count:value",
        'An approximate count of rows in this table' as "stats:row_count:description",
        (row_count is not null) as "stats:row_count:include",

        'Approximate Size' as "stats:bytes:label",
        bytes as "stats:bytes:value",
        'Approximate size of the table as reported by Snowflake' as "stats:bytes:description",
        (bytes is not null) as "stats:bytes:include",

        'Last Modified' as "stats:last_modified:label",
        to_varchar(convert_timezone('UTC', last_altered), 'yyyy-mm-dd HH24:MI'||'UTC') as "stats:last_modified:value",
        'The timestamp for last update/change' as "stats:last_modified:description",
        (last_altered is not null and table_type='BASE TABLE') as "stats:last_modified:include"
    from HEVO_DB.INFORMATION_SCHEMA.tables
            where (
                (
                    
    "table_schema" ilike 'HEVO_SCHEMA' and upper("table_schema") = upper('HEVO_SCHEMA')

                    and 
    "table_name" ilike 'raw_payments' and upper("table_name") = upper('raw_payments')

                )
             or 
                (
                    
    "table_schema" ilike 'HEVO_SCHEMA' and upper("table_schema") = upper('HEVO_SCHEMA')

                    and 
    "table_name" ilike 'raw_customers' and upper("table_name") = upper('raw_customers')

                )
             or 
                (
                    
    "table_schema" ilike 'HEVO_SCHEM' and upper("table_schema") = upper('HEVO_SCHEM')

                    and 
    "table_name" ilike 'customer_aggregation' and upper("table_name") = upper('customer_aggregation')

                )
             or 
                (
                    
    "table_schema" ilike 'HEVO_SCHEMA' and upper("table_schema") = upper('HEVO_SCHEMA')

                    and 
    "table_name" ilike 'raw_orders' and upper("table_name") = upper('raw_orders')

                )
            )
        ),
        columns as (
            select
        table_catalog as "table_database",
        table_schema as "table_schema",
        table_name as "table_name",

        column_name as "column_name",
        ordinal_position as "column_index",
        data_type as "column_type",
        comment as "column_comment"
    from HEVO_DB.INFORMATION_SCHEMA.columns
            where (
                (
                    
    "table_schema" ilike 'HEVO_SCHEMA' and upper("table_schema") = upper('HEVO_SCHEMA')

                    and 
    "table_name" ilike 'raw_payments' and upper("table_name") = upper('raw_payments')

                )
             or 
                (
                    
    "table_schema" ilike 'HEVO_SCHEMA' and upper("table_schema") = upper('HEVO_SCHEMA')

                    and 
    "table_name" ilike 'raw_customers' and upper("table_name") = upper('raw_customers')

                )
             or 
                (
                    
    "table_schema" ilike 'HEVO_SCHEM' and upper("table_schema") = upper('HEVO_SCHEM')

                    and 
    "table_name" ilike 'customer_aggregation' and upper("table_name") = upper('customer_aggregation')

                )
             or 
                (
                    
    "table_schema" ilike 'HEVO_SCHEMA' and upper("table_schema") = upper('HEVO_SCHEMA')

                    and 
    "table_name" ilike 'raw_orders' and upper("table_name") = upper('raw_orders')

                )
            )
        )
        select *
    from tables
    join columns using ("table_database", "table_schema", "table_name")
    order by "column_index"
[0m15:25:41.451913 [debug] [ThreadPool]: Opening a new connection, currently in state init
[0m15:25:45.843204 [debug] [ThreadPool]: SQL status: SUCCESS 30 in 4.391 seconds
[0m15:25:45.856314 [debug] [ThreadPool]: On HEVO_DB.information_schema: Close
[0m15:25:46.170984 [info ] [MainThread]: Catalog written to /home/ubuntu/HEVO_POC_STEPS/target/catalog.json
[0m15:25:46.172876 [debug] [MainThread]: Resource report: {"command_name": "generate", "command_success": true, "command_wall_clock_time": 6.859558, "process_user_time": 2.711844, "process_kernel_time": 0.189661, "process_mem_max_rss": "124140", "process_out_blocks": "4816", "process_in_blocks": "0"}
[0m15:25:46.173573 [debug] [MainThread]: Command `dbt docs generate` succeeded at 15:25:46.173445 after 6.86 seconds
[0m15:25:46.173937 [debug] [MainThread]: Connection 'generate_catalog' was properly closed.
[0m15:25:46.174292 [debug] [MainThread]: Connection 'HEVO_DB.information_schema' was properly closed.
[0m15:25:46.174650 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x714ccdab5af0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x714ccde45c70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x714ccdeabc80>]}
[0m15:25:46.175078 [debug] [MainThread]: Flushing usage events
[0m15:25:53.213837 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7218155ec4d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x721815614140>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7218156927e0>]}


============================== 15:25:53.217041 | 8f6ee443-b078-4b76-a924-d596a0cfbde5 ==============================
[0m15:25:53.217041 [info ] [MainThread]: Running with dbt=1.8.7
[0m15:25:53.217675 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': '/home/ubuntu/HEVO_POC_STEPS/logs', 'profiles_dir': '/home/ubuntu/.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt docs serve', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m15:25:53.688352 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'project_id', 'label': '8f6ee443-b078-4b76-a924-d596a0cfbde5', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x721813d83aa0>]}
[0m15:25:53.750222 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'adapter_info', 'label': '8f6ee443-b078-4b76-a924-d596a0cfbde5', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x721813fb8500>]}
[0m15:38:36.026046 [error] [MainThread]: Encountered an error:

[0m15:38:36.031949 [error] [MainThread]: Traceback (most recent call last):
  File "/home/ubuntu/dbt-env/lib/python3.12/site-packages/dbt/cli/requires.py", line 138, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/dbt-env/lib/python3.12/site-packages/dbt/cli/requires.py", line 101, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/dbt-env/lib/python3.12/site-packages/dbt/cli/requires.py", line 218, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/dbt-env/lib/python3.12/site-packages/dbt/cli/requires.py", line 247, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/dbt-env/lib/python3.12/site-packages/dbt/cli/requires.py", line 294, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/dbt-env/lib/python3.12/site-packages/dbt/cli/main.py", line 303, in docs_serve
    results = task.run()
              ^^^^^^^^^^
  File "/home/ubuntu/dbt-env/lib/python3.12/site-packages/dbt/task/docs/serve.py", line 29, in run
    httpd.serve_forever()
  File "/usr/lib/python3.12/socketserver.py", line 235, in serve_forever
    ready = selector.select(poll_interval)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/selectors.py", line 415, in select
    fd_event_list = self._selector.poll(timeout)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt

[0m15:38:36.033209 [debug] [MainThread]: Resource report: {"command_name": "serve", "command_wall_clock_time": 762.8751, "process_user_time": 2.078809, "process_kernel_time": 0.181534, "process_mem_max_rss": "111292", "process_in_blocks": "208", "process_out_blocks": "2968", "command_success": false}
[0m15:38:36.033948 [debug] [MainThread]: Command `dbt docs serve` failed at 15:38:36.033811 after 762.88 seconds
[0m15:38:36.034587 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7218155ec4d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x721813d80470>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x721813c9e690>]}
[0m15:38:36.034962 [debug] [MainThread]: Flushing usage events
[0m15:38:54.056044 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7cd701df3a70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7cd701ed0bf0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7cd701df38f0>]}


============================== 15:38:54.059046 | e9e719a4-de22-4645-b329-8f3ea6f36103 ==============================
[0m15:38:54.059046 [info ] [MainThread]: Running with dbt=1.8.7
[0m15:38:54.059604 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': '/home/ubuntu/HEVO_POC_STEPS/logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt docs serve', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m15:38:54.525256 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'project_id', 'label': 'e9e719a4-de22-4645-b329-8f3ea6f36103', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7cd700726f60>]}
[0m15:38:54.593346 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'adapter_info', 'label': 'e9e719a4-de22-4645-b329-8f3ea6f36103', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7cd703246ae0>]}
[0m16:41:41.861774 [error] [MainThread]: Encountered an error:

[0m16:41:41.866582 [error] [MainThread]: Traceback (most recent call last):
  File "/home/ubuntu/dbt-env/lib/python3.12/site-packages/dbt/cli/requires.py", line 138, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/dbt-env/lib/python3.12/site-packages/dbt/cli/requires.py", line 101, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/dbt-env/lib/python3.12/site-packages/dbt/cli/requires.py", line 218, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/dbt-env/lib/python3.12/site-packages/dbt/cli/requires.py", line 247, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/dbt-env/lib/python3.12/site-packages/dbt/cli/requires.py", line 294, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/dbt-env/lib/python3.12/site-packages/dbt/cli/main.py", line 303, in docs_serve
    results = task.run()
              ^^^^^^^^^^
  File "/home/ubuntu/dbt-env/lib/python3.12/site-packages/dbt/task/docs/serve.py", line 29, in run
    httpd.serve_forever()
  File "/usr/lib/python3.12/socketserver.py", line 235, in serve_forever
    ready = selector.select(poll_interval)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/selectors.py", line 415, in select
    fd_event_list = self._selector.poll(timeout)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt

[0m16:41:41.868963 [debug] [MainThread]: Resource report: {"command_name": "serve", "command_wall_clock_time": 3767.8647, "process_user_time": 2.355805, "process_kernel_time": 0.277138, "process_mem_max_rss": "111088", "process_in_blocks": "32", "process_out_blocks": "2968", "command_success": false}
[0m16:41:41.870262 [debug] [MainThread]: Command `dbt docs serve` failed at 16:41:41.870124 after 3767.87 seconds
[0m16:41:41.870667 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7cd70401cc50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7cd700a0d2e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7cd7006902c0>]}
[0m16:41:41.871088 [debug] [MainThread]: Flushing usage events
